templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>

```

Comment:
The links in the sidebar are placeholders (`#`). Consider updating these to point to the actual URLs of the respective pages in the application.